Sources/Sentry/SentryWatchdogTerminationScopeObserver.m

```diff
@@ -49,7 +49,7 @@ - (void)clearBreadcrumbs
 
 - (void)setContext:(nullable NSDictionary<NSString *, id> *)context
 {
-    [self.contextProcessor setContext:context];
+    [self.attributesProcessor setContext:context];
```

Comment:
`m`: Summarizing all processor into a single `attributesProcessor` kind of defeats the purpose of the processor classes spreading logic into multiple types. It's fine to stay this way.